backend/.env.example

```
@@ -1,187 +1,76 @@
 
 # -----------------------------------------------------------------------------
 # GEMINI LLM CONFIGURATION (Alternative LLM Provider)
 # -----------------------------------------------------------------------------
-# Get your API key from: https://aistudio.google.com/app/apikey
-# Optional: Alternative to OpenAI for story generation
-GEMINI_LLM_API_KEY=your_gemini_llm_api_key_here
-GEMINI_LLM_MODEL=gemini-2.0-flash-thinking-exp-1219
-GEMINI_LLM_ENDPOINT=https://generativelanguage.googleapis.com/v1beta/models
+GEMINI_LLM_API_KEY=
+GEMINI_LLM_MODEL=gemini-2.5-flash-lite
+GEMINI_LLM_ENDPOINT=https://generativelanguage.googleapis.com/v1beta/openai/models
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GEMINI_LLM_ENDPOINT key should go before the GEMINI_LLM_MODEL key

(UnorderedKey)